src/main/java/com/client/core/security/tools/rsa/GenerateRSAKeyPair.java

```diff
@@ -0,0 +1,50 @@
+package com.client.core.security.tools.rsa;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.security.*;
+
+/**
+ * Created by kcable on 6/2/2017.
+ *
+ * Used to create the public and private keys for rsa encryption.
+ */
+public class GenerateRSAKeyPair {
```

Comment:
Should this be included? Makes me wonder why it wasn't added earlier.

Reply:
This might have been something that I grabbed from `cs-starter-template` on accident

Reply:
@taswartz In case you're not doing it already, can we cleanup cs-starter-template stuff that might have been incorporatred here on accident?

Reply:
@scriptom 
Yes! I just removed these 2 files and looking for more that might have snuck in